static/app/components/nav/config.tsx

```diff
@@ -0,0 +1,191 @@
+import {useMemo} from 'react';
+
+import type {NavItemRaw, SidebarItem, SubmenuItem} from 'sentry/components/nav/utils';
+import {
+  getNavigationItemStatus,
+  NAV_DIVIDER,
+  resolveSidebarItem,
+  splitAtDivider,
+} from 'sentry/components/nav/utils';
+import {
+  IconDashboard,
+  IconGraph,
+  IconIssues,
+  IconLightning,
+  IconProject,
+  IconSearch,
+  IconSettings,
+  IconSiren,
+} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {getDiscoverLandingUrl} from 'sentry/utils/discover/urls';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useModuleURLBuilder} from 'sentry/views/insights/common/utils/useModuleURL';
+import {MODULE_SIDEBAR_TITLE} from 'sentry/views/insights/http/settings';
+import {MODULE_TITLES} from 'sentry/views/insights/settings';
+import {getSearchForIssueGroup, IssueGroup} from 'sentry/views/issueList/utils';
+
+export interface NavItemsResult {
+  primary: {
+    body: ReadonlyArray<SidebarItem>;
+    footer: ReadonlyArray<SidebarItem>;
+  };
+  secondary: {
+    body: ReadonlyArray<SidebarItem>;
+    footer: ReadonlyArray<SidebarItem>;
+  };
+}
+
+export function useNavItems(): NavItemsResult {
+  const organization = useOrganization();
+  const location = useLocation();
+  const moduleURLBuilder = useModuleURLBuilder(true);
+  const prefix = `organizations/${organization.slug}`;
+
+  const items = useMemo<NavItemRaw[]>(
+    () => [
+      {
+        label: t('Issues'),
+        icon: <IconIssues />,
+        submenu: [
+          {
+            label: t('All'),
+            to: `/${prefix}/issues/?query=is:unresolved`,
+          },
+          {
+            label: t('Error & Outage'),
+            to: `/${prefix}/issues/${getSearchForIssueGroup(IssueGroup.ERROR_OUTAGE)}`,
+          },
+          {
+            label: t('Trend'),
+            to: `/${prefix}/issues/${getSearchForIssueGroup(IssueGroup.TREND)}`,
+          },
+          {
+            label: t('Craftsmanship'),
+            to: `/${prefix}/issues/${getSearchForIssueGroup(IssueGroup.CRAFTSMANSHIP)}`,
+          },
+          {
+            label: t('Security'),
+            to: `/${prefix}/issues/${getSearchForIssueGroup(IssueGroup.SECURITY)}`,
+          },
+          {label: t('Feedback'), to: `/${prefix}/feedback/`},
+        ],
+      },
+      {label: t('Projects'), to: `/${prefix}/projects/`, icon: <IconProject />},
+      {
+        label: t('Explore'),
+        icon: <IconSearch />,
+        submenu: [
+          {
+            label: t('Traces'),
+            to: `/traces/`,
```

Comment:
Why doesn't this have the prefix?

Reply:
Great callout, better to use the standard `Feature` interface! Resolved in [a9c0f1a](https://github.com/getsentry/sentry/pull/77665/commits/a9c0f1aac5788b60053556463c6669a4bbcf4fe7)